sorc/grid_tools.fd/regional_esg_grid.fd/pmat4.f90

```diff
@@ -1732,31 +1904,28 @@ subroutine znfun(n,z,zn,znd,zndd,znddd)!                               [znfun]
 endif
 end subroutine znfun
       
-!=============================================================================
-! Utility code for various Mobius transformations. If aa1,bb1,cc1,dd1 are
-! the coefficients for one transformation, and aa2,bb2,cc2,dd2 are the 
-! coefficients for a second one, then the coefficients for the mapping
-! of a test point, zz, by aa1 etc to zw, followed by a mapping of zw, by
-! aa2 etc to zv, is equivalent to a single mapping zz-->zv by the transformatn
-! with coefficients aa3,bb3,cc3,dd3, such that, as 2*2 complex matrices:
-!
-! [ aa3, bb3 ]   [ aa2, bb2 ]   [ aa1, bb1 ]
-! [          ] = [          ] * [          ]
-! [ cc3, dd3 ]   [ cc2, dd2 ]   [ cc1, dd1 ] .
-!
-!  Note that the determinant of these matrices is always +1
-!
-!=============================================================================
+!> Utility code for various Mobius transformations. If aa1,bb1,cc1,dd1 are
+!! the coefficients for one transformation, and aa2,bb2,cc2,dd2 are the 
+!! coefficients for a second one, then the coefficients for the mapping
+!! of a test point, zz, by aa1 etc to zw, followed by a mapping of zw, by
+!! aa2 etc to zv, is equivalent to a single mapping zz-->zv by the transformatn
+!! with coefficients aa3,bb3,cc3,dd3, such that, as 2*2 complex matrices:
+!!
+!! [ aa3, bb3 ]   [ aa2, bb2 ]   [ aa1, bb1 ]
+!! [          ] = [          ] * [          ]
+!! [ cc3, dd3 ]   [ cc2, dd2 ]   [ cc1, dd1 ] .
+!!
+!!  Note that the determinant of these matrices is always +1
+!!
+!! @param[in] v matric
```

Comment:
Missing params.

Reply:
The original document does not detail the parameter z and infz. What should I put in here?